modules/users/user.js

```diff
@@ -44,6 +53,46 @@ describe("DHIS2 API - Users Module", function () {
         });
     });
 
+    describe("Authenticate User", function () {
+        it("should authenticate updated User using new credentials", function () {
+            var response = chakram.get(env.url + "/api/me", utils.buildParams(data.testUserUpdated));
+            expect(response).to.have.status(200);
+            return chakram.wait();
+        });
+
+        it("should fail authentication for updated User using old credentials", function () {
+            var response = chakram.get(env.url + "/api/me", utils.buildParams(data.testUser));
+            expect(response).to.have.status(401);
+            expect(response).to.have.header("content-type", "text/html;charset=utf-8");
+            return chakram.wait();
+        });
+    });
+
+    describe("Update User", function () {
+        var response;
+
+        before(function () {
+            response = chakram.get(env.url + "/api/users?query=pgracio", env.auth);
+            return response;
+        });
+
+        it("should not be possible to update the password with an invalid password (less than 8 characters).", function () {
```

Comment:
@jason-p-pickering apparently password policy is not being verified. Would expect a 400 bad request.